### ISSUE TITLE
is this project completely implement the result of the paper?

### ISSUE THREAD

**Issue by xyy19920105** [yufengm/Adaptive#2]
is this project completely implement the result of the paper?

**Comment by yufengm** [comment]
Yes. Except I'm building upon my own provided dataset and I didn't implement all the visualization part in the original paper.

**Comment by yufengm** [comment]
Not really. I haven't implemented the beam search part in the paper.

**Comment by TaihuLight** [comment]
Image-Captioning using InceptionV3 and Beam Search
https://github.com/yashk2810/Image-Captioning

**Comment by yufengm** [comment]
Thanks.
